File: src/main/java/kitchenpos/eatinorder/tobe/domain/OrderTableRepository.java
package kitchenpos.eatinorder.tobe.domain;

import kitchenpos.eatinorder.tobe.domain.OrderTable;
import java.util.List;
import java.util.Optional;
import java.util.UUID;

public interface OrderTableRepository {
    kitchenpos.eatinorder.domain.OrderTable save(OrderTable orderTable);

    Optional<OrderTable> findById(UUID id);

    List<OrderTable> findAll();
}

File: src/main/kotlin/kitchenpos/order/tobe/eatinorder/domain/OrderTableRepository.kt
package kitchenpos.order.tobe.eatinorder.domain

import java.util.*
import org.springframework.context.annotation.Primary
import org.springframework.stereotype.Repository

@Repository("tobeOrderTableRepository")
@Primary
interface OrderTableRepository {
    fun save(orderTable: OrderTable): OrderTable

    fun findById(id: UUID): Optional<OrderTable>

    fun findAll(): List<OrderTable>
}

